doc/node_config_json_schema.json

```diff
@@ -0,0 +1,609 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "debug-brk": {
+      "type": "boolean",
+      "required": []
```

Comment:
What are all these `required` fields? I thought this was only relevant to the `object` type.

Reply:
yep removed them